ui-components/src/headers/SiteHeader.tsx

```diff
@@ -66,24 +67,26 @@ const SiteHeader = (props: SiteHeaderProps) => {
 
   const logo = (logoClass = "") => {
     return (
-      <LocalizedLink
-        className={`navbar-item logo ${logoClass} ${getLogoWidthClass()}`}
-        href="/"
-        aria-label="homepage"
-      >
-        <div
-          className={`logo__lockup ${getLogoWidthClass()} ${
-            props.logoWidth && "navbar-custom-width"
-          } ${props.imageOnly && "navbar-image-only-container"}`}
+      <h2>
```

Comment:
Does this component always go in the `<h2>` place? I'm wondering if it makes more sense to put the `<h2>` outside of this component so that it can be instantiated in other contexts.

Reply:
I think the site header should always be h2 assuming we're making the document title h1. 

I do have to check some other components that also appear in the application flow before contributing upstream, because the heading order might need to be different there.

Reply:
Maybe Exygy will have feedback when you post it upstream.